platform/ios/src/MGLAnnotationView.h

```diff
@@ -89,7 +89,7 @@ typedef NS_ENUM(NSUInteger, MGLAnnotationViewDragState) {
  contains a non-`nil` value while the annotation view is visible on the map. If
  the view is queued, waiting to be reused, the value is `nil`.
  */
-@property (nonatomic, readonly, nullable) id <MGLAnnotation> annotation;
+@property (nonatomic, nullable) id <MGLAnnotation> annotation;
```

Comment:
Even though `MKAnnotationView.annotation` is read-write, it also comes with this note in the documentation:

> You should not change the value of this property directly.

which we also state above, redundantly. What is our reason for making the property read-write but advising developers not to write to it?